src/Sql/Sql.Test/ScenarioTests/DatabaseReplicationTests.ps1

```diff
@@ -197,6 +197,40 @@ function Test-CreateNamedSecondaryDatabase()
 	}
 }
 
+<#
+	.SYNOPSIS
+	Tests creating a named secondary database of already existing database
+#>
+function Test-CreateNamedSecondaryDatabaseNegative()
```

Comment:
Is this the only negative test? In case we add more negative tests, can we be more specific like Test-CreateNamedSecondaryAlreadyExists or something similar to indicate the test scenario?